packages/survey-creator-core/src/property-grid/index.ts

```diff
@@ -1281,22 +1274,22 @@ export abstract class PropertyGridEditor implements IPropertyGridEditor {
     question: Question,
     options: ISurveyCreatorOptions,
     onClose?: (reason: "apply" | "cancel") => void
-  ): IPropertyEditorSetup {
```

Comment:
Just reverting this line and the others in this function should fix your unit tests @andrewtelnov. (I guess, have no easy way to confirm)